### ISSUE TITLE
startup screen makes easysession-load fail

### ISSUE THREAD

**Issue by linwaytin** [jamescherti/easysession.el#19]
If the startup screen buffer exists, the `easysession-load` fails. I need to kill it and then run `easysession-load`. Note that if `easysession-load` has been run and failed, it will fail afterward. I think this is a separate bug.

Workaround:
```elisp
(setq inhibit-startup-screen t)
```

**Comment by jamescherti** [comment]
Thank you for reporting this issue and using . Unfortunately, I couldn't reproduce the issue.  

Could you provide a minimal `init.el` configuration that reproduces the problem? This will help me investigate further.

**Comment by jamescherti** [comment]
I will close this ticket. If the issue persists, feel free to reopen it.

**Comment by linwaytin** [comment]
Yes, sorry that I don't have time to investigate this issue at this moment. It might be related to the daemon mode of Emacs, but I'm not sure and I do not use it recently.
